src/cloud/social/alias/brorand.js

```diff
@@ -0,0 +1,18 @@
+// brorand alias which works in a freedomjs module context.
+// Uses randombytes, for which we have an alias which works
+// for Firefox.
+
+const randombytes = require('randombytes');
+
+module.exports = function rand(len) {
```

Comment:
I've gotta say this is a very strange module to my eyes at least, though I get what it's doing - I'd probably add a "TODO - remove when Firefox has better webworker randomness", just for peace of mind.